mlshvch/hometask-5/2.sql

```diff
@@ -0,0 +1,5 @@
+SELECT f.title, l.name
+FROM film f
```

Comment:
Isn't it much easier to just write "film" instead of random letter? Let's avoid unnecessary shortening and keep the code as much readable as possible.
Applies to the rest of files as well.